### DISCUSSION TITLE
mini.diff keymaps not functioning properly.

### DISCUSSION THREAD

**Discussion by redoxahmii** [nvim-mini/mini.nvim#1403] [Q&A] [answered]
It's a weird issue that I've had for a while but never had the time to investigate properly.
I use LazyVim and with the default provided for navigating hunks with `]h` and `[h` i find it a bit awkward to use so i had keymaps that use `<c-j>` and `<c-k>`.
Issue is when I initially open a buffer and use it it works fine but upon navigating to  to neo-tree window and back it stops working after that.
I suspect this might not be something specific to `mini.diff` but i still wanted to ask here as some more people might've had similar issues.

Thank you!

**Comment by redoxahmii** [comment, marked as answer]
It was `vim-tmux-navigator` disabling it has made it work.
I really wish I could both these at the same time but unfortunately not.
If you have a workaround do tell

**Reply by echasnovski** [reply]
> If you have a workaround do tell

Sorry, but the only suggestion I have is to use different keys which do not conflict with one another.

The default `[h` and `]h` are in line with common `[` and `]` meaning of "navigate to previous/next" something.

**Reply by redoxahmii** [reply]
yeah i figured.
I only find the hunks navigation a bit awkward as you have to extend your hand a little for `]` and use the same hand finger for pressing `h` then which slows it down a bit if you're trying to navigate quickly through hunks but i believe i have quite large hands so it's skill issue on my part lol.
I should try to use the other hand which might be a good solution.
Thank you for your plugins!
